tools/generators/legacy/src/Generator/Environment.swift

```diff
@@ -24,6 +24,7 @@ struct Environment {
     let consolidateTargets: (
         _ targets: [TargetID: Target],
         _ xcodeGeneratedFiles: [FilePath: FilePath],
+        _ targetNameMode: TargetNameMode,
```

Comment:
After the above change, we should be able to remove it from here.

Reply:
Fixed